lib/cell/railtie.rb

```diff
@@ -14,6 +14,9 @@ class Railtie < Rails::Railtie
 
     # ruthlessly stolen from the zurb-foundation gem.
     initializer "cells.update_asset_paths" do |app|
+      # Add Rails.root to view_paths
+      Cell::ViewModel.view_paths = [Rails.root.join("app", "cells")]
```

Comment:
engines already need to set the path like specified on docs `view_paths = ["#{MyEngine::Engine.root}/app/concepts"]`

this change has no direct impact on engine cells